packages/minifiers/minifiers.js

```diff
@@ -109,7 +109,7 @@ var rewriteRules = function (rules) {
     // a folder starting with "/packages/" is served on the same path as
     // it was on the filesystem and everything else is served on root "/".
     if (! basePath.match(/^\/?packages\//i))
-        basePath = "/";
+        basePath = (process.env.ROOT_URL_PATH_PREFIX || "") + "/";
```

Comment:
I understand that, but I didn't see a change in this PR that said "in order to use this feature, set the ROOT_URL_PATH_PREFIX env var".